runtime/libpgmath/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ if(${LIBPGMATH_SYSTEM_PROCESSOR} MATCHES "aarch64" AND NOT ${LIBPGMATH_WITH_GENE
   else()
     message(STATUS "Libpgmath CPU target explicitly set to ${LLVM_FLANG_CPU_TARGET}")
   endif()
-  if(FLANG_ENABLE_QUADFP)
+  if(FLANG_ENABLE_QUADFP AND NOT ${TARGET_OS} STREQUAL "Windows")
```

Comment:
Arm test cases are now failing in precommit. Could it be that this logic has caused that? Maybe it should be put into parentheses, e.g. `((FLANG_ENABLE_QUADFP) AND NOT (${TARGET_OS} STREQUAL "Windows"))`

Reply:
Thanks for your comment. Hope I will work for now. That was my fault, I should have used LIBPGMATH_SYSTEM_PROCESSOR instead of TARGET_OS.

Reply:
NB why this variable is named `LIBPGMATH_SYSTEM_PROCESSOR`? It's so confusing, if it summarizes CPU and OS, it should be named simply `LIBPGMATH_PLATFORM`.

Reply:
nah, I looked into other parts of pgmath and found this:
```
elseif(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
```
I guess you should have used `(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")` instead...